src/ui/flutter_app/lib/shared/theme/app_theme.dart

```diff
@@ -111,36 +73,111 @@ class AppTheme {
     valueIndicatorTextStyle: TextStyle(fontSize: 24),
   );
 
+  static const _cardTheme = CardTheme(
+    margin: EdgeInsets.symmetric(vertical: 4.0),
+    color: _cardColor,
+  );
+
+  static final _textTheme = TextTheme(
+    bodyText1: TextStyle(
+      fontSize: LocalDatabaseService.display.fontSize,
+    ),
+  );
+
   static TextStyle simpleDialogOptionTextStyle = TextStyle(
     fontSize: LocalDatabaseService.display.fontSize + 4.0,
-    color: AppTheme.simpleDialogOptionTextColor,
+    color: _simpleDialogOptionTextColor,
   );
 
   static TextStyle moveHistoryTextStyle = TextStyle(
     fontSize: LocalDatabaseService.display.fontSize + 2.0,
     height: 1.5,
-    color: moveHistoryTextColor,
+    color: _moveHistoryTextColor,
   );
 
-  static double boardTop = isLargeScreen ? 75.0 : 36.0;
-  static double boardMargin = 10.0;
-  static double boardScreenPaddingH = 10.0;
-  static double boardBorderRadius = 5.0;
-  static double boardPadding = 5.0;
+  static TextStyle drawerHeaderTextStyle = TextStyle(
+    fontSize: LocalDatabaseService.display.fontSize + 16,
+    fontWeight: FontWeight.w600,
+  );
 
-  static TextStyle settingsHeaderStyle = TextStyle(
-    color: settingsHeaderTextColor,
+  static TextStyle dialogTitleTextStyle = TextStyle(
     fontSize: LocalDatabaseService.display.fontSize + 4,
+    color: _appPrimaryColor,
   );
 
-  static TextStyle settingsTextStyle =
-      TextStyle(fontSize: LocalDatabaseService.display.fontSize);
+  static const TextStyle copyrightTextStyle = TextStyle(
+    fontSize: _copyrightFontSize,
+  );
 
-  static const cardMargin = EdgeInsets.symmetric(vertical: 4.0);
+  static final TextStyle notationTextStyle = TextStyle(
+    fontSize: 20,
+    color: LocalDatabaseService.colorSettings.boardLineColor,
+  );
 
-  static const double drawerWidth = 250.0;
+  static const listTileSubtitleStyle = TextStyle(
+    color: listTileSubtitleColor,
+  );
+
+  static const listTileTitleStyle = TextStyle(
+    color: _switchListTileTitleColor,
+  );
+
+  static final mainToolbarTextStyle = TextStyle(
+    color: LocalDatabaseService.colorSettings.mainToolbarIconColor,
+  );
+
+  static const helpTextStyle = TextStyle(
+    color: helpTextColor,
+  );
+
+  static const licenseTextStyle = TextStyle(
+    fontFamily: 'Monospace',
+    fontSize: 12,
+  );
+
+  static const double boardMargin = 10.0;
+  static const double boardScreenPaddingH = 10.0;
+  static const double boardBorderRadius = 5.0;
+  static const double boardPadding = 5.0;
+
+  static TextStyle settingsHeaderStyle = TextStyle(
+    color: _settingsHeaderTextColor,
+    fontSize: LocalDatabaseService.display.fontSize + 4,
+  );
+
+  static TextStyle settingsTextStyle = TextStyle(
+    fontSize: LocalDatabaseService.display.fontSize,
+  );
 
   static const double sizedBoxHeight = 16.0;
 
-  static double copyrightFontSize = 12;
+  static const double _copyrightFontSize = 12;
+
+  /// Game page
```

Comment:
Why are some variables preceded by an underscore and some variables are not?

Reply:
The underscore in dart is somewhat comparable with the `private` annotation in java. 
Not all attributes can be made "private" because some are accessed outside the theme class.